docs/README.md

```diff
@@ -22,47 +22,29 @@ A list of downloadable CLIs for various operating systems, and CPU architectures
 location `{BASE URL}/cli/go/download`. The `{BASE URL}` is the OpenWhisk API hostname or IP address
 (e.g., openwhisk.ng.bluemix.net).
 
-There are three properties to configure the CLI with:
+There are two required properties to configure in order to use the CLI:
 
 1. **API host** (name or IP address) for the OpenWhisk deployment you want to use.
 2. **Authorization key** (username and password) which grants you access to the OpenWhisk API.
-3. **Namespace** where your OpenWhisk assets are stored.
 
-The CLI will usually have an API host already set. You can check its value with
-`wsk property get --apihost`.
-
-If you know your authorization key and namespace, you can configure the CLI to use them. Otherwise
-you will need to provide one or both for most CLI operations.
+The API host can be acquired from the `edge.host` property in `whisk.properties` file, which is generated during
```

Comment:
I think the API host is dependent on the environment in which the deployment is happening.